target/linux/lantiq/dts/DM200.dts

```diff
@@ -0,0 +1,208 @@
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+
+    model = "DM200 - Netgear DM200";
+
+    chosen {
+        bootargs = "console=ttyLTQ0,115200";
+
+    };
+
+    aliases {
+        led-boot = &power_green;
+        led-failsafe = &power_amber;
+        led-running = &power_green;
+
+        led-internet = &dsl_green;
+        led-dsl = &dsl_amber;
+    };
+
+    memory@0 {
+        reg = <0x0 0x4000000>;
+    };
+
+    fpi@10000000 {
+        gpio: pinmux@E100B10 {
+            pinctrl-names = "default";
+            pinctrl-0 = <&state_default>;
+
+            state_default: pinmux {
+                mdio {
+                    lantiq,groups = "mdio";
+                    lantiq,function = "mdio";
+                };
+                conf_out {
+                    lantiq,pins = "io12", "io15";  // annexa, annexb
```

Comment:
Hmm, this commend is from the vendor dts, but I'm not sure it's correct. On my connection (VDSL2, annex b), removing the io15 pin had no effect, while removing the io12 pin caused the connection to not come up.